testfiles/5ZoneSwimmingPool.idf

```diff
@@ -3513,20 +3513,20 @@
 
   Output:Variable,*,Indoor Pool Current Activity Factor,hourly;
 
-  Output:Variable,*,Indoor Pool Current Cover Factor,hourly;
-
+  Output:Variable,*,Indoor Pool Current Cover Factor,hourly;

```

Comment:
Hmm, what's going on here?  I can't see the diff.  Invalid character maybe?  Time to crack open the ol' hex editor.